src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx

```diff
@@ -254,9 +254,9 @@ For more about how you'd assign roles to groups and create custom roles, see the
 
 A role, whether one of our [standard roles](#standard-roles) or a custom role, is a set of capabilities. To view roles and their associated capabilities, use the [**Organization and access** UI](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks#where).
 
-![New Relic capabilities UI screenshot](./images/New-Relic-capabilities-UI-screenshot.png "New-Relic-capabilities-UI-screenshot.png")
+![New Relic user capabilities UI screenshot](./images/new-relic-user-capabilities-ui.png "New Relic user capabilities screenshot")
```

Comment:
@zuluecho9, this image is difficult for me to see. It's fine when I blow it up 200% but I have a huge monitor. It sounds like you want the screenshot to be a legal document that shows the access options at a moment in time. Is there a programatic way to scrape this data from the UI and put it into a series of tables in the docs?